### PR TITLE
minor cleanup to the template proposal

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@bgrant0607 fixes the brace issue and a couple other minor things i noticed when reviewing it.

File: docs/proposals/templates.md [hunk 1]
```diff
@@ -395,7 +394,8 @@ pod template.
 * **/processedtemplates** - when a template is POSTed to this endpoint, all parameters in the template are processed and
 substituted into appropriate locations in the object definitions.  Validation is performed to ensure required parameters have
 a value supplied.  In addition labels defined in the template are applied to the object definitions.  Finally the customized
-template (still a `Template` object) is returned to the caller.
+template (still a `Template` object) is returned to the caller.  (The possibility of returning an ObjectList instead has
```
Comment (bgrant0607) [inline docs/proposals/templates.md]: s/ObjectList/List/

File: docs/proposals/templates.md [hunk 2]
```diff
@@ -437,10 +437,11 @@ The api endpoint will then:
     of the `$(PARAM)` parameter.
   * If a given $(VAL) could be resolved to either a parameter or an environment variable/downward api reference, an error will be
     returned.
-5. Return the processed template object.
+5. Return the processed template object. (or ObjectList, depending on the choice made when this is implemented)
```
Comment (bgrant0607) [inline docs/proposals/templates.md]: s/ObjectList/List/

File: docs/proposals/templates.md [hunk 3]
```diff
@@ -437,10 +437,11 @@ The api endpoint will then:
...
 
 The client can now either return the processed template to the user in a desired form (eg json or yaml), or directly iterate the
-api objects within the template, invoking the appropriate object creation api endpoint for each element.
+api objects within the template, invoking the appropriate object creation api endpoint for each element.  (If the api returns
+an ObjectList, the client would simply iterate the list to create the objects).
```
Comment (bgrant0607) [inline docs/proposals/templates.md]: ditto

**Comment by bgrant0607** [comment]
Thanks.

**Comment by bparees** [comment]
@bgrant0607 fixed.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**